app/forms.py

```diff
@@ -32,7 +32,19 @@
     DATE_QUALIFIER_CHOICES
 )
 
-
+GENDER_CHOICES=[
+    ('Male','Male'),
+    ('Female','Female'),
+    ('Other', 'Other')
+    ]
+IDENTITY_CHOICES = [
+    ('', 'Select Identity'),
+    ('First Nations Status', 'First Nations Status'),
```

Comment:
Consider using lowercase for all options with no options. It makes working with the data in the db easier.
e.g `('first_nations_status', 'First Nations Status')`